usr/src/cmd/zdb/zdb.c

```diff
@@ -2142,6 +2175,117 @@ dump_label_uberblocks(vdev_label_t *lbl, uint64_t ashift)
 }
 
 static int
+dump_specific_path(objset_t *os, uint64_t obj)
+{
+	int print_header = 1;
+
+	dump_object(os, obj, dump_opt['v'], &print_header);
+
+	return (0);
+}
+
+static int
+dump_specific_dir(objset_t *os, uint64_t obj, char *topname, char *name)
```

Comment:
can `topname` be declared `const`?  looks like it's only used for error reporting.